CRM/Admin/Form/PaymentProcessor.php

```diff
@@ -184,6 +184,9 @@ public function buildQuickForm($check = FALSE) {
       CRM_Financial_BAO_PaymentProcessor::buildOptions('payment_processor_type_id'),
       TRUE
     );
+    if ($this->_action !== 1) {
```

Comment:
Is not immediately clear to me what this `1` means. Should this be a constant?

Reply:
```suggestion
    if ($this->_action !== CRM_Core_Action::ADD) {
```

Reply:
Arg, yeah. I had spent a lot of time in`*.tpl` files where their convention was using mysterious numbers. The constant is better. :+1: